Map/Test/MapTest.cpp

```diff
@@ -269,3 +269,10 @@ TEST_F(SquareMapTest, seenDoorsAndWallsShouldStayVisible)
 
     ASSERT_EQ(expectedNumberOfVisibleFields, numberOfVisibleFields);
 }
+
+TEST_F(SquareMapTest, mapPrint)
+{
+	SquareMap map(2);
+	
+	ASSERT_EQ(map.getMapToPrint(Position(1,1)),"    1 2\n  #######\n1 # . F #\n2 # T P #\n  #######");
```

Comment:
Nie jestem pewny czy sie da, ale sporbuj to tak podzielic z uzyciem /

"    1 2\n/
  #######\n/
1 # . F #\n/
2 # T P #\n/
  #######"

żeby bylo czytelniej. Nie pamietam czy to bylo /, ale jakos można "lamac" tekst"